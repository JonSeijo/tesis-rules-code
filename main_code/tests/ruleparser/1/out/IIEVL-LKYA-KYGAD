{IIEVL,LKYA}=>{KYGAD}

tests/ruleparser/1/in/proteins/p2.fasta,SDLGKKLLEAARAGQDDEVRILMANGADVNANDWFGITPLHLVVNNGHLEIIEVLLKYAADVNASDKSGWTPLHLAAYRGHLEIVEVLLKYGADVNAMDYQGYTPLHLAAEDGHLEIVEVLLKYGADVNAQDKFGKTAFDISIDNGNEDLAEILQ
