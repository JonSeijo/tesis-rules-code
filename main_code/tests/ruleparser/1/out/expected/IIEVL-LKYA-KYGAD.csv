{IIEVL,LKYA}=>{KYGAD}
p2.fasta,SDLGKKLLEAARAGQDDEVRILMANGADVNANDWFGITPLHLVVNNGHLEIIEVLLKYAADVNASDKSGWTPLHLAAYRGHLEIVEVLLKYGADVNAMDYQGYTPLHLAAEDGHLEIVEVLLKYGADVNAQDKFGKTAFDISIDNGNEDLAEILQ
